.env.example

```diff
@@ -6,47 +7,58 @@ APP_URL=http://localhost
 
 LOG_CHANNEL=stack
 
+# Change this according to your database configuration
+# To use SQLite, have a look here: https://laravel.com/docs/8.x/database#sqlite-configuration1
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
 DB_DATABASE=homestead
 DB_USERNAME=homestead
 DB_PASSWORD=secret
 
+# These are the recommended default values
 BROADCAST_DRIVER=log
 CACHE_DRIVER=file
 QUEUE_CONNECTION=sync
 SESSION_LIFETIME=120
 
+# To be able to send mails and debug them, you can set up a mailtrap.io account and enter your credentials here
+# You can also take a look here for other configuration possibilities: https://laravel.com/docs/8.x/mail#configuration
 MAIL_DRIVER=smtp
 MAIL_HOST=smtp.mailtrap.io
 MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
 
-# The derhuerst/db-rest:v5 instance which we use.
-# You can create your own instance
-DB_REST=http://localhost:3000
+# This is the official derhuerst/db-rest:v5 instance. Please set up your own if you're planning on using this more than
+# once in a while so their servers are not being overloaded.
+DB_REST=https://v5.db.transport.rest
 
-# Prevent TRWL from posting to socialMedia
+# When set to TRUE, Träwelling will not attempt to post to social media, even if the users want to.
 POST_SOCIAL=FALSE
 
+# Twitter app credentials, only needed for logging in with and posting to twitter
+# https://developer.twitter.com/en/apply-for-access
 TWITTER_REDIRECT=http://localhost:8000/callback/twitter
 TWITTER_ID=
 TWITTER_SECRET=
 
+# Mastodon app credentials, only needed for logging in with and posting to mastodon
+# https://docs.joinmastodon.org/methods/apps/
 MASTODON_DOMAIN=
 MASTODON_ID=
 MASTODON_SECRET=
 MASTODON_REDIRECT=
 MASTODON_APPNAME=
 
+# Default values to be shown in the imprint. Especially important in Germany.
```

Comment:
```suggestion
# Default values to be shown in the legal notice.
```
I think it is not only important in germany. also "imprint" is not the best wording. I know, we are using it as a title there, but we should change this in the future to "legal notice" or something.

Reply:
I think as long as the value is called "imprint_XXX" we should be consistent here.